op-node/rollup/driver/state.go

```diff
@@ -266,20 +284,18 @@ func (s *Driver) eventLoop() {
 
 		select {
 		case <-sequencerCh:
-			payload, err := s.sequencer.RunNextSequencerAction(ctx)
-			if err != nil {
-				s.log.Error("Sequencer critical error", "err", err)
+			if err := sequencerStep(); err != nil {
 				return
 			}
-			if s.network != nil && payload != nil {
-				// Publishing of unsafe data via p2p is optional.
-				// Errors are not severe enough to change/halt sequencing but should be logged and metered.
-				if err := s.network.PublishL2Payload(ctx, payload); err != nil {
-					s.log.Warn("failed to publish newly created block", "id", payload.ID(), "err", err)
-					s.metrics.RecordPublishingError()
-				}
+			continue
+		default:
+		}
```

Comment:
Yeah, the core improvement should be done in op-geth👍.  As to this pr, it changes the schedule of sequencerStep and otherSteps(e.g. derivateStep...):
<img width="767" alt="image" src="https://github.com/ethereum-optimism/optimism/assets/33241113/d1b3cfc6-f12c-4fc7-88a7-852b9cb014d7">
* former schedule, there is chance derivateStep hit the block-produce-timestamp rather than sequencerStep
* after schedule, more likely to let sequencerStep hit the block-produce-timestamp, and arrange the rest steps(e.g. derivateStep) to the rest of time window

This may be a little optimization i think.